### PR TITLE
Added acceptable altitude range to move and hover.

### PR DESCRIPTION
**Pull request by Thomas-McKanna**
Added a constant for ACCEPTABLE_ALTITUDE_DEVIATION (set at 0.1 for now) which allows movement tasks and hover tasks to only correct movement if altitude has drifted beyond the acceptable range (in this case 0.1 meters from the target altitude). 

I also added the capability of setting and altitude for movement tasks, since previously it always went to the default altitude, which might not be want we want 100% of the time.

Also added some documentation to LinearMovementTask that was missing from before.

Also adjusted the PID values so that corrections that do happen are not so severe (I think this was contributing the the "hopping" behavior we saw in real-life tests).

### CODE REVIEW

**Review by ClayMav — APPROVED** [review]

File: flight/tasks/hover_task.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from task_base import TaskBase
 from .. import constants as c
 
-KP = 1
```
Comment (christopher-o-toole) [inline flight/tasks/hover_task.py:6]: Again, these constants should be documented.

**Review by christopher-o-toole — CHANGES_REQUESTED** [review]
I think this is a good improvement relative to our previous method of control, however you should document  the PID gain value constants to make it more readable to those less familiar with the flight code.

**Review by ClayMav — CHANGES_REQUESTED** [review]
Just what Chris said

File: flight/tasks/hover_task.py (hunk 1 above)
Comment (Thomas-McKanna) [reply flight/tasks/hover_task.py:6]: I fixed this issue in the `feature/imports` pull request.

**Review by christopher-o-toole — APPROVED** [review]

**Review by ClayMav — APPROVED** [review]

**Comment by ClayMav** [comment]
Lets get this merged @Thomas-McKanna

**Pull request merged**